CHANGELOG.md

```diff
@@ -528,6 +528,15 @@
 - Drag and drop to reorder tiles @robgietema
 - Enhanced DraftJS AnchorLink Plugin @robgietema @sneridagh
 - Added the configuration required in Webpack config to load CSS modules in the project, required by DraftJS AnchorLink plugin @sneridagh
+* Refactor UserControlPanel add delete and add user actions @csenger
```

Comment:
@nileshgulia1 This does not seem right. Can you re-add your changelog to the latest unreleased block on the changelog?

Reply:
@tisto I have added the changelog!

Reply:
@nileshgulia1 you still have to remove those duplicate changelog entries here.

Reply:
Sorry. I removed the old entries now.